librad/src/meta/entity.rs

```diff
@@ -145,50 +146,12 @@ pub trait VerificationStatusMarker {
     fn status() -> VerificationStatus;
 }
 
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub struct EntityStatusVerified {
-    marker: (),
-}
-impl EntityStatusVerified {
-    pub(self) fn new() -> Self {
-        Self { marker: () }
-    }
-}
-impl VerificationStatusMarker for EntityStatusVerified {
-    fn status() -> VerificationStatus {
-        VerificationStatus::Verified
-    }
-}
-
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub struct EntityStatusSigned {
-    marker: (),
-}
-impl EntityStatusSigned {
-    pub(self) fn new() -> Self {
-        Self { marker: () }
-    }
-}
-impl VerificationStatusMarker for EntityStatusSigned {
-    fn status() -> VerificationStatus {
-        VerificationStatus::Signed
-    }
-}
-
-#[derive(Clone, Debug, PartialEq, Eq)]
-pub struct EntityStatusUnknown {
-    marker: (),
-}
-impl EntityStatusUnknown {
-    pub(self) fn new() -> Self {
-        Self { marker: () }
-    }
-}
-impl VerificationStatusMarker for EntityStatusUnknown {
-    fn status() -> VerificationStatus {
-        VerificationStatus::Unknown
-    }
-}
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Verified;
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Signed;
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Since we are already renaming types, maybe we could rename `Unknown` as `Draft`?